numpy/distutils/command/build_ext.py

```diff
@@ -586,6 +586,13 @@ def build_extension(self, ext):
             # not using fcompiler linker
             self._libs_with_msvc_and_fortran(
                 fcompiler, libraries, library_dirs)
+            if ext.runtime_library_dirs:
+                # gcc adds RPATH to the link. On windows, copy the dll into
+                # self.extra_dll_dir instead.
+                for d in ext.runtime_library_dirs:
+                    for f in glob(d + '/*.dll'):
+                        copy_file(f, self.extra_dll_dir)
+                ext.runtime_library_dirs = []
```

Comment:
This change is connected to the use of a DLL directly from the OpenBLAS build, rather than pulling out the `libopenblas.a` and wrapping it via gfortran to create another DLL. Apparently numpy/distutils was never set up to do this before.